wpilibc/sim/src/RobotDrive.cpp

```diff
@@ -20,7 +20,8 @@ using namespace frc;
 
 const int RobotDrive::kMaxNumberOfMotors;
 
-static auto make_shared_nodelete(SpeedController* ptr) {
+static std::shared_ptr<SpeedController> make_shared_nodelete(
```

Comment:
Actually its defnitely not that. That fails to build because its returning a ref to a temporary